phe/paillier.py

```diff
@@ -87,8 +88,9 @@ def __init__(self, n):
         self.max_int = n // 3 - 1
 
     def __repr__(self):
-        nsquare = self.nsquare.to_bytes(1024, 'big')
-        g = self.g.to_bytes(1024, 'big')
+        digest_bytes = max(1024, round(math.log2(self.nsquare)))
```

Comment:
Ok I'll make some changes. What I really want to do is represent `g` and `nsquare` in the smallest 2**n number of bytes with a min of 1024.